go/pkg/pwagent/daemon.go

```diff
@@ -57,17 +64,57 @@ func Daemon(ctx context.Context, cli *client.Client, logger *zap.Logger) error {
 	fmt.Println("api instances", godev.PrettyJSON(apiInstances))
 
 	// fetch local info from docker daemon
-	dockerInstances, err := pwcompose.GetPathwarInfo(ctx, cli)
+	pathwarInfo, err := pwcompose.GetPathwarInfo(ctx, cli)
```

Comment:
if you find something more accurate than `pathwarInfo` and `pwcompose.GetPathwarInfo`, I would love it <3

Reply:
kk will think about it maybe open an issue for a future refactor ?